docs/contribute/create-diagram.md

```diff
@@ -0,0 +1,111 @@
+  [rename and organize](https://help.figma.com/hc/en-us/articles/360038663994-Name-and-organize-components) elements.
+
+!!! tip "Tips"
+
+    - Hold down ++cmd++ on Mac or ++ctrl++ on Windows to
+      [select](https://help.figma.com/hc/en-us/articles/360040449873-Select-layers-and-objects) elements excluding the
+      frame.
+    - Hold down ++opt++ and drag to duplicate an element.
+
+### Style
+
+- Use the premade diagram assets as starting points.
+  By default, you can adjust the width of the premade labels, but the height is automatically sized to the number of
+  lines of text.
+  To freely customize a component, right-click on it, **detach instance**, and **remove auto layout**.
+- Use the pre-defined [color styles](https://help.figma.com/hc/en-us/articles/360039820134-Manage-and-share-styles) or
+  black (#00000).
+- Use rounded corners of radius 2 for rectangular labels and containers.
+- Evenly [align](https://help.figma.com/hc/en-us/articles/360039956914-Adjust-alignment-rotation-and-position) elements.
+- Represent similar conceptual elements using similar styles.
+  For example, represent two nodes using a dark gradient, and represent two external components using a light gradient.
+- You can reuse existing icons from any diagram on any page.
+  For example, there are already icons to represent databases, dapps, keys, locks, and logos.
+
+!!! tip
+
+    Hold down ++shift++ when drawing, resizing, and rotating to create perfect horizontal and vertical lines.
+
+#### Text
+
+- Use font **Roboto** for all text.
+- Use font sizes between 10-18.
+- Use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/capitalization) in labels and titles.
+
+#### Arrows and lines
+
+- Use a thickness of 2 for arrows, lines, borders, and other strokes.
+- Use **Triangle** arrow heads.
+- Use straight arrows and lines, with right-angle bends if needed.
+  Don't use diagonal arrows and lines.
+  If possible, don't overlap arrows and lines.
+  To create additional bends in an arrow or line, **detach instance** (if applicable), double-click the arrow or line,
+  and click and drag the anchor points.
+- Leave about 3px of space between arrow heads and the elements they point to.
+  Line ends without arrow heads should touch the connecting element.
+
+!!! example
+
+    ![Example diagram](../assets/images/besu-tessera-high-availability.png)
+
+See the
+[Figma documentation on the Arrow Tool](https://help.figma.com/hc/en-us/articles/360040450133-Using-Shape-Tools#h_677f8eba-73c4-4987-a64b-c0226aaec392)
+for more information.
+
+### Export
+
+To export your diagram:
+
+1. Select the frame of your diagram.
+    Make sure all elements of your diagram are contained in the frame.
+    The name of this frame will be the name of the exported image.
+
+1. Scroll to the bottom of the right sidebar.
+    In the **Export** section, choose **2x** scale (for retina screens) and **PNG** or **JPG** file format.
```

Comment:
Note that if you want to keep your image small (in terms of kb) and still keep the best quality whatever the size it will be displayed (on a big screen or on a mobile phone), you can have it exported in SVG vector format. It's probably even better than PNG for diagrams.